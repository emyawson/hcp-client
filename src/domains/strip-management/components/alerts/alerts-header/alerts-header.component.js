import React from 'react';

import { GridItemWithoutVerticalPadding, LocalizedText } from 'src/components';

import {
  AlertsInternalGrid,
  AlertsPageHeader,
  AlertsHeaderTitleP,
} from '../alerts.style';

export const AlertsHeader = () => (
  <AlertsPageHeader>
    <AlertsInternalGrid>
      <GridItemWithoutVerticalPadding span="1" columns="4">
        <AlertsHeaderTitleP>
          <LocalizedText textKey="alerts.preIdealInterval" />
        </AlertsHeaderTitleP>
      </GridItemWithoutVerticalPadding>
      <GridItemWithoutVerticalPadding span="1" columns="4">
        <AlertsHeaderTitleP>
          <LocalizedText textKey="alerts.postIdealInterval" />
        </AlertsHeaderTitleP>
      </GridItemWithoutVerticalPadding>
      <GridItemWithoutVerticalPadding span="1" columns="4">
        <AlertsHeaderTitleP>
          <LocalizedText textKey="alerts.noctIdealInterval" />
        </AlertsHeaderTitleP>
      </GridItemWithoutVerticalPadding>
      <GridItemWithoutVerticalPadding span="1" columns="4">
        <AlertsHeaderTitleP>
          <LocalizedText textKey="alerts.alertSettings" />
        </AlertsHeaderTitleP>
      </GridItemWithoutVerticalPadding>
    </AlertsInternalGrid>
  </AlertsPageHeader>
);
