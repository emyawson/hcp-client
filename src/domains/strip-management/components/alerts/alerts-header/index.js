export * from './alerts-header.component';
