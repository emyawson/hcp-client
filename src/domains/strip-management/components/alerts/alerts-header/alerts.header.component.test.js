import React from 'react';
import { shallow } from 'enzyme';

import { AlertsHeader } from './alerts-header.component';

describe('Alerts header component', () => {
  it('Renders correctly', () => {
    const wrapper = shallow(<AlertsHeader />);
    expect(wrapper).toMatchSnapshot();
  });
});
