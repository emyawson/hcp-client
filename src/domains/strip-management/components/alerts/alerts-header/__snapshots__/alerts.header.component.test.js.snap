// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alerts header component Renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "class",
    "props": Object {
      "children": <styled.div>
        <GridItemWithoutVerticalPadding
          columns="4"
          span="1"
        >
          <styled.p>
            <LocalizedText
              textKey="alerts.preIdealInterval"
            />
          </styled.p>
        </GridItemWithoutVerticalPadding>
        <GridItemWithoutVerticalPadding
          columns="4"
          span="1"
        >
          <styled.p>
            <LocalizedText
              textKey="alerts.postIdealInterval"
            />
          </styled.p>
        </GridItemWithoutVerticalPadding>
        <GridItemWithoutVerticalPadding
          columns="4"
          span="1"
        >
          <styled.p>
            <LocalizedText
              textKey="alerts.noctIdealInterval"
            />
          </styled.p>
        </GridItemWithoutVerticalPadding>
        <GridItemWithoutVerticalPadding
          columns="4"
          span="1"
        >
          <styled.p>
            <LocalizedText
              textKey="alerts.alertSettings"
            />
          </styled.p>
        </GridItemWithoutVerticalPadding>
      </styled.div>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": null,
      "nodeType": "class",
      "props": Object {
        "children": Array [
          <GridItemWithoutVerticalPadding
            columns="4"
            span="1"
          >
            <styled.p>
              <LocalizedText
                textKey="alerts.preIdealInterval"
              />
            </styled.p>
          </GridItemWithoutVerticalPadding>,
          <GridItemWithoutVerticalPadding
            columns="4"
            span="1"
          >
            <styled.p>
              <LocalizedText
                textKey="alerts.postIdealInterval"
              />
            </styled.p>
          </GridItemWithoutVerticalPadding>,
          <GridItemWithoutVerticalPadding
            columns="4"
            span="1"
          >
            <styled.p>
              <LocalizedText
                textKey="alerts.noctIdealInterval"
              />
            </styled.p>
          </GridItemWithoutVerticalPadding>,
          <GridItemWithoutVerticalPadding
            columns="4"
            span="1"
          >
            <styled.p>
              <LocalizedText
                textKey="alerts.alertSettings"
              />
            </styled.p>
          </GridItemWithoutVerticalPadding>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": null,
          "nodeType": "class",
          "props": Object {
            "children": <styled.p>
              <LocalizedText
                textKey="alerts.preIdealInterval"
              />
            </styled.p>,
            "columns": "4",
            "span": "1",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": null,
            "nodeType": "class",
            "props": Object {
              "children": <LocalizedText
                textKey="alerts.preIdealInterval"
              />,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": null,
              "nodeType": "function",
              "props": Object {
                "textKey": "alerts.preIdealInterval",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "type": [Function],
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": null,
          "nodeType": "class",
          "props": Object {
            "children": <styled.p>
              <LocalizedText
                textKey="alerts.postIdealInterval"
              />
            </styled.p>,
            "columns": "4",
            "span": "1",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": null,
            "nodeType": "class",
            "props": Object {
              "children": <LocalizedText
                textKey="alerts.postIdealInterval"
              />,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": null,
              "nodeType": "function",
              "props": Object {
                "textKey": "alerts.postIdealInterval",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "type": [Function],
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": null,
          "nodeType": "class",
          "props": Object {
            "children": <styled.p>
              <LocalizedText
                textKey="alerts.noctIdealInterval"
              />
            </styled.p>,
            "columns": "4",
            "span": "1",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": null,
            "nodeType": "class",
            "props": Object {
              "children": <LocalizedText
                textKey="alerts.noctIdealInterval"
              />,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": null,
              "nodeType": "function",
              "props": Object {
                "textKey": "alerts.noctIdealInterval",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "type": [Function],
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": null,
          "nodeType": "class",
          "props": Object {
            "children": <styled.p>
              <LocalizedText
                textKey="alerts.alertSettings"
              />
            </styled.p>,
            "columns": "4",
            "span": "1",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": null,
            "nodeType": "class",
            "props": Object {
              "children": <LocalizedText
                textKey="alerts.alertSettings"
              />,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": null,
              "nodeType": "function",
              "props": Object {
                "textKey": "alerts.alertSettings",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "type": [Function],
          },
          "type": [Function],
        },
      ],
      "type": [Function],
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "class",
      "props": Object {
        "children": <styled.div>
          <GridItemWithoutVerticalPadding
            columns="4"
            span="1"
          >
            <styled.p>
              <LocalizedText
                textKey="alerts.preIdealInterval"
              />
            </styled.p>
          </GridItemWithoutVerticalPadding>
          <GridItemWithoutVerticalPadding
            columns="4"
            span="1"
          >
            <styled.p>
              <LocalizedText
                textKey="alerts.postIdealInterval"
              />
            </styled.p>
          </GridItemWithoutVerticalPadding>
          <GridItemWithoutVerticalPadding
            columns="4"
            span="1"
          >
            <styled.p>
              <LocalizedText
                textKey="alerts.noctIdealInterval"
              />
            </styled.p>
          </GridItemWithoutVerticalPadding>
          <GridItemWithoutVerticalPadding
            columns="4"
            span="1"
          >
            <styled.p>
              <LocalizedText
                textKey="alerts.alertSettings"
              />
            </styled.p>
          </GridItemWithoutVerticalPadding>
        </styled.div>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": null,
        "nodeType": "class",
        "props": Object {
          "children": Array [
            <GridItemWithoutVerticalPadding
              columns="4"
              span="1"
            >
              <styled.p>
                <LocalizedText
                  textKey="alerts.preIdealInterval"
                />
              </styled.p>
            </GridItemWithoutVerticalPadding>,
            <GridItemWithoutVerticalPadding
              columns="4"
              span="1"
            >
              <styled.p>
                <LocalizedText
                  textKey="alerts.postIdealInterval"
                />
              </styled.p>
            </GridItemWithoutVerticalPadding>,
            <GridItemWithoutVerticalPadding
              columns="4"
              span="1"
            >
              <styled.p>
                <LocalizedText
                  textKey="alerts.noctIdealInterval"
                />
              </styled.p>
            </GridItemWithoutVerticalPadding>,
            <GridItemWithoutVerticalPadding
              columns="4"
              span="1"
            >
              <styled.p>
                <LocalizedText
                  textKey="alerts.alertSettings"
                />
              </styled.p>
            </GridItemWithoutVerticalPadding>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": null,
            "nodeType": "class",
            "props": Object {
              "children": <styled.p>
                <LocalizedText
                  textKey="alerts.preIdealInterval"
                />
              </styled.p>,
              "columns": "4",
              "span": "1",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": null,
              "nodeType": "class",
              "props": Object {
                "children": <LocalizedText
                  textKey="alerts.preIdealInterval"
                />,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": null,
                "nodeType": "function",
                "props": Object {
                  "textKey": "alerts.preIdealInterval",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              "type": [Function],
            },
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": null,
            "nodeType": "class",
            "props": Object {
              "children": <styled.p>
                <LocalizedText
                  textKey="alerts.postIdealInterval"
                />
              </styled.p>,
              "columns": "4",
              "span": "1",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": null,
              "nodeType": "class",
              "props": Object {
                "children": <LocalizedText
                  textKey="alerts.postIdealInterval"
                />,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": null,
                "nodeType": "function",
                "props": Object {
                  "textKey": "alerts.postIdealInterval",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              "type": [Function],
            },
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": null,
            "nodeType": "class",
            "props": Object {
              "children": <styled.p>
                <LocalizedText
                  textKey="alerts.noctIdealInterval"
                />
              </styled.p>,
              "columns": "4",
              "span": "1",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": null,
              "nodeType": "class",
              "props": Object {
                "children": <LocalizedText
                  textKey="alerts.noctIdealInterval"
                />,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": null,
                "nodeType": "function",
                "props": Object {
                  "textKey": "alerts.noctIdealInterval",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              "type": [Function],
            },
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": null,
            "nodeType": "class",
            "props": Object {
              "children": <styled.p>
                <LocalizedText
                  textKey="alerts.alertSettings"
                />
              </styled.p>,
              "columns": "4",
              "span": "1",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": null,
              "nodeType": "class",
              "props": Object {
                "children": <LocalizedText
                  textKey="alerts.alertSettings"
                />,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": null,
                "nodeType": "function",
                "props": Object {
                  "textKey": "alerts.alertSettings",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              "type": [Function],
            },
            "type": [Function],
          },
        ],
        "type": [Function],
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
